src/components/Practice/AudioTooltipModal.jsx

```
@@ -37,16 +51,26 @@ const AudioTooltipModal = ({ audioSrc, description, children }) => {
+  };
+
   return (
     <div
+      ref={containerRef}
       style={{ position: "relative", display: "inline-block" }}
-      onMouseEnter={() => setShowModal(true)}
-      onMouseLeave={() => setShowModal(false)}
+      onMouseEnter={() => !isTouchDevice && setShowModal(true)}
+      onMouseLeave={() => !isTouchDevice && setShowModal(false)}
+      onClick={handleClick}
     >
```

ESLint

[error] Visible, non-interactive elements with click handlers must have at least one keyboard listener.

(jsx-a11y/click-events-have-key-events)